src/restapi/mm_routes.ts

```diff
@@ -0,0 +1,49 @@
+import fs from 'fs';
+
+import express, {Request, Response} from 'express';
+import {AppCall} from 'mattermost-redux/types/apps';
+
+import app from '../app/app';
+
+import {jsonStoreFileName} from '../app/constants';
+
+const router = express.Router();
+
+router.post('/createform', async (req: Request, res: Response) => {
+    const appCall: AppCall = req.body;
+    if (appCall.type === 'form') {
+        const createForm = app.calls.createForm(appCall.context.post.message);
+        res.json(createForm);
+    } else {
+        const message = app.createTicketFromPost(appCall);
+        res.json({});
+    }
+});
+
+router.get('/manifest.json', (req: Request, res: Response) => {
+    res.json(app.manifest);
+});
+
+router.get('/bindings', (req: Request, res: Response) => {
+    res.json(app.bindings.getBindings());
+});
+
+router.post('/install', (req: Request, res: Response) => {
+    // Write json file store
+    fs.writeFile(jsonStoreFileName, JSON.stringify(req.body.values), 'utf8', (err) => {
```

Comment:
Can this functionality be put into a function in another file, like `storeInstallInfo`?

Reply:
I added this as method in the `app/store.ts`, inside the JSONFileStore class

Reply:
done -> https://github.com/mattermost/mattermost-app-zendesk/commit/fa8a3e3c7bcebec08dd689fbae3927ceb05f7847